tests/singlerod/long/openmc_heat_surrogate/enrico.xml

```diff
@@ -14,10 +14,11 @@
     <clad_outer_radius>0.475</clad_outer_radius>
     <fuel_rings>20</fuel_rings>
     <clad_rings>5</clad_rings>
-    <pin_centers>
-      0. 0.
-    </pin_centers>
+    <n_pins_x>1</n_pins_x>
+    <n_pins_y>1</n_pins_y>
+    <pin_pitch>1.1</pin_pitch>
+    <mass_flowrate>1.0</mass_flowrate>
+    <inlet_temperature>350.0</inlet_temperature>
```

Comment:
If you're wondering, I think this is what's causing the test to fail. The OpenMC model starts with a uniform temperature of 523 K and loads cross sections between 300 and 1500 K, but the cross section data we're using has temperatures of 294 K, 500 K, 750 K, etc., so the lowest point that is actually loaded is 500 K. Thus, when one of the temperatures gets set somewhere close to 350 K, it crashes when trying to load data there (definitely not great behavior from OpenMC to just segfault -- I'll file an issue there). If you raise the inlet to > 500 K, that should fix the problem.

Reply:
Hmm, yeah I didn't see any failures on my machine, so it must be how the test is set up. Thanks for explaining!